### PR TITLE
Add VARTA Storage integration 

### PR DESCRIPTION
**Pull request by Vip0r**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/63736
- Link to parent pull request in the Brands repository: 
https://github.com/home-assistant/brands/pull/3049

- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/vartastorage.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@Vip0r'
+ha_domain: vartastorage
+ha_platforms:
+  - sensor
+---
+
+[VARTA Storage](https://www.varta-storage-portal.com) allows you to integrate your Battery device into HomeAssistant.
```
Comment (frenck) [inline source/_integrations/vartastorage.markdown]:
```suggestion
[VARTA Storage](https://www.varta-storage-portal.com) allows you to integrate your Battery device into Home Assistant.
```

File: source/_integrations/vartastorage.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+There is currently support for the following device types within Home Assistant:
+
+- Sensor
+
+The integration will create sensor entities for the different values of the device like State of Charge, Charge Power, Power Grid Consumption and many more.
+Setup of this integrations requires that your battery device is connected to your LAN and you know the actual IP Address/Hostname of it.
+The Port ist pre-defined with 502 (Standard Modbus Port). The integration is using this [PyPI Package](https://github.com/vip0r/vartastorage)
```
Comment (frenck) [inline source/_integrations/vartastorage.markdown]:
```suggestion
The integration will create sensor entities for the different values of the device like the State of Charge, Charge Power, Power Grid Consumption, and many more.

# Prerequisites

To set up this integration, it is required that your battery device is connected to your network.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Vip0r** [comment]
Updated the filename of this integration-doc as recommended within the PR for the integration

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the linked parent PR has been closed.

**Pull request closed without merging**